scripts/o3de/o3de/export_project.py

```diff
@@ -7,28 +7,69 @@
 #
 
 import argparse
+import fnmatch
+import glob
 import logging
 import os
 import pathlib
 import platform
-import sys
+import psutil
+import shutil
 
 from o3de import manifest, utils
+from typing import List
+
+# Account for some windows-specific attributes
+if platform.system().lower() == 'windows':
+    EXECUTABLE_EXTENSION = '.exe'
+    O3DE_SCRIPT_NAME = 'o3de.bat'
+    GENERATOR = None
+    CMAKE_GENERATOR_OPTIONS = ['-DLY_DISABLE_TEST_MODULES=ON']
+else:
+    EXECUTABLE_EXTENSION = ""
+    O3DE_SCRIPT_NAME = 'o3de.sh'
+    GENERATOR = "Ninja Multi-Config"
```

Comment:
Should we always auto assign Ninja generator for unix? There may be use cases where Unix Makefiles are preferred (is there an option to set/override the default generator?)

Reply:
In the android setup scripts, it tries to auto-detect if ninja is available before using it or falling back to unix make files. I can add that kind of logic here as well